templates/webapps/galaxy/tool_form.mako

```diff
@@ -3,455 +3,22 @@
 ${h.js("libs/bibtex", "libs/jquery/jquery-ui")}
 ${h.css('base', 'jquery-ui/smoothness/jquery-ui')}
 
-## skip new tool form code if disabled
-%if util.string_as_bool(trans.app.config.get('toolform_upgrade',  True)):
```

Comment:
Yes, this disables the old tool form for good. Currently we are building both tool forms in parallel. Since sections are not supported in the old tool form, we can either integrate them or disable the old tool form for good which would allow us to clean up the code and remove the associated makos all together. However, as John pointed out the latter has the consequence that the upload and ftp tool parameters will be deprecated.